fetch.bs

```diff
@@ -194,8 +194,76 @@ lt="authentication entry">authentication entries</a> (for HTTP authentication).
 
  <dt><dfn for="fetch params">task destination</dfn> (default null)
  <dd>Null, a <a for=/>global object</a>, or a <a for=/>parallel queue</a>.
+ <dt><dfn for="fetch params">timing info</dfn>
+ <dd>A <a for=/>fetch timing info</a>.
+ <dt><dfn for="fetch params">redirect timing info list</dfn>(default an empty list)
+ <dd>A list of <a for=/>redirect timing info</a>
 </dl>
 
+<p>A <dfn export>fetch timing info</dfn> is a <a for=/>struct</a> used to maintain timing
+information later required by the resource timing and navigation timing specs. It has the following
+<a for=struct>items</a>:
+<dl>
+  <dt><dfn export for="fetch timing info">fetch start time</dfn> (default zero)
+  <dt><dfn export for="fetch timing info">worker start time</dfn> (default zero)
+  <dt><dfn export for="fetch timing info">request start time</dfn> (default zero)
+  <dt><dfn export for="fetch timing info">response start time</dfn> (default zero)
+  <dt><dfn export for="fetch timing info">response end time</dfn> (default zero)
+  <dd>A <a for=/>DOMHighResTimeStamp</a>.
+  <dt><dfn export for="fetch timing info">encoded body size</dfn> (default zero)
+  <dt><dfn export for="fetch timing info">decoded body size</dfn> (default zero)
+  <dd>A number.
+  <dt><dfn export for="fetch timing info">connection timing info</dfn> (default null)
+  <dd>Null or a <a for=/>connection timing info</a>.
+</dl>
+
+<p>A <dfn export>redirect timing info</dfn> is a <a for=/>struct</a> used to maintain timing
+information about a redirect.
+<a for=struct>items</a>:
+<dl>
+  <dt><dfn export for="redirect timing info">redirect location URL</dfn>
```

Comment:
Why do we need to record this in two places? And why is it needed to begin with?